src/sim/seq.rs

```diff
@@ -120,15 +118,6 @@ where
             }
 
             self.iter_limit.inc();
-            self.duration = match self.duration {
```

Comment:
Rather than remove this code, I would refactor it to a free function (not exported) and have that function have a different implementation based on the target. The regular implementation would be this removed code, while the WASM implementation would simply return `None`.

Could you also tell me how you compiled this project for WASM (I tried building it and got over 90 errors for undefined identifiers, such as `c_long`)?

Reply:
I'll get something up soon that demonstrates that

Reply:
about the Instant stuff, it could be as easy as an if/else Option each time, I just thought that there might be a better idea :laughing: I guess we can improve later on as well though

Reply:
> I'll get something up soon that demonstrates that

Thanks :+1:. That will allow me to help you better.

> about the Instant stuff

I don't really understand what you mean here, but we can take a look at it once I get your PR to compile

Reply:
I successfully build it like that:
~~~
cargo +nightly build --target wasm32-unknown-unknown
wasm-bindgen target/wasm32-unknown-unknown/release/wasm_tspsolver.wasm  --nodejs
~~~